XenAdmin/MainWindow.cs

```diff
@@ -2666,16 +2666,27 @@ private void Updates_CollectionChanged(CollectionChangeEventArgs e)
 
         private void UpdatesCheck_Completed(bool suceeded, string err)
         {
-            
-            Program.Invoke(this, () => {
+
+            Program.Invoke(this, () =>
+            {
                 updateAlert = Updates.UpdateAlerts.FirstOrDefault(update => update is ClientUpdateAlert) as ClientUpdateAlert;
-                updateClientToolStripMenuItem.Enabled = updateAlert != null; 
+                DisableAndHideUpdateClientToolStripMenuItem(updateAlert != null);

```

Comment:
Could we check for updates on startup? Currently, if there are no updates, the update client button is shown. Then it disappears once I manually check for updates.

If we don't want to check for updates on startup, we should at least hide the button by default in my opinion

Reply:
this might be a missed setting on initial creation that check at startup as we do check at astart up